decidim-initiatives/app/cells/decidim/initiatives/content_blocks/highlighted_initiatives_cell.rb

```diff
@@ -3,37 +3,36 @@
 module Decidim
   module Initiatives
     module ContentBlocks
-      class HighlightedInitiativesCell < Decidim::ViewModel
-        include Decidim::SanitizeHelper
+      class HighlightedInitiativesCell < Decidim::ContentBlocks::HighlightedParticipatorySpacesCell
+        BLOCK_ID = "highlighted-initiatives"
 
         delegate :current_organization, to: :controller
 
-        def show
-          render if highlighted_initiatives.any?
+        def highlighted_spaces
+          @highlighted_spaces ||= OrganizationPrioritizedInitiatives
+                                  .new(current_organization, order)
+                                  .query
         end
 
-        def max_results
-          model.settings.max_results
+        def i18n_scope
+          "decidim.initiatives.pages.home.highlighted_initiatives"
         end
 
-        def order
-          model.settings.order
+        def all_path
+          Decidim::Initiatives::Engine.routes.url_helpers.initiatives_path
         end
 
-        def highlighted_initiatives
-          @highlighted_initiatives ||= OrganizationPrioritizedInitiatives
-                                       .new(current_organization, order)
-                                       .query
-                                       .limit(max_results)
-        end
+        private
 
-        def i18n_scope
-          "decidim.initiatives.pages.home.highlighted_initiatives"
+        def max_results
+          model.settings.max_results
         end
 
-        def decidim_initiatives
-          Decidim::Initiatives::Engine.routes.url_helpers
+        def order
+          model.settings.order
         end
+
+        def block_id = BLOCK_ID
```

Comment:
Same comment as in [highlighted_assemblies_cell.rb](https://github.com/decidim/decidim/pull/10920/files#diff-b3db5c57a4634c36562d7145ffcab03aebd4955fa796a9daa5b031fac9bcdb9e)